### PR TITLE
Made the Proxy usage reports script more decent

### PR DESCRIPTION
**Pull request by uaalto**

### CODE REVIEW

File: etc/device_tx_analysis.py [hunk 1]
```diff
@@ -17,111 +18,129 @@
...
+        retrieve_redis = misc_util.confirm("Do you want to update the Redis file?")
+
+    if retrieve_redis:
+        print "Retrieving data from Redis..."
+        data = redis_shell.zrange('client->bytesInOut', 0, -1, withscores=True)
+        with open("dump.csv", "wb") as f:
+            writer = csv.writer(f)
+            writer.writerows(data)
+
+    rev = reversed([r for r in clean_data()])
```
Comment (uaalto) [inline etc/device_tx_analysis.py]: This actually didn't work without the list comprehension. Might be because clean_data is a generator (is it?)

**Comment by uaalto** [comment]
Not urgent @aranhoide, but is fairly straightforward

File: etc/device_tx_analysis.py (hunk 1 above)
Comment (aranhoide) [inline etc/device_tx_analysis.py]: Sorry, reversed takes a sequence.  So you can say `reversed(list(clean_data()))`.

**Comment by aranhoide** [comment]
LGTM (I guess you only wanted to ask the reverse thing?).

**Pull request merged**